bin/check-instance-events.rb

```diff
@@ -66,55 +66,68 @@ def aws_config
       region: config[:aws_region] }
   end
 
+  def ec2_regions
+    # This is for SDK v2
+    # Aws.partition('aws').regions.map(&:name)
+
+    AWS::EC2.regions.map(&:name)
+  end
+
   def run
     event_instances = []
     aws_config = {}
 
+    aws_regions = ec2_regions
+
+    unless config[:aws_region].casecmp('all') == 0
```

Comment:
@sstarcher before I make fixes to this PR, I'd like to gain some info on where are we heading towards in terms of re-factoring code to use aws sdk v2. 

Yes, you added support for AWS_REGION env var, but your common code is based on aws sdk v2. 

Whereas the scripts this PR is updating are based on aws sdk v1, which are not based on (and wouldn't work with) your 'common' lib. 

Should I re-factor the scripts to be based on sdk v2 as well?

Thanks